site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -289,26 +273,27 @@ Reload the extension to view the green button.
 ## Layer logic {: #logic }
 
 The extension now knows the popup should be available to users on [developer.chrome.com][23] and
-displays a colored button, but needs logic for further user interaction. Update `popup.js` to
-include the following code.
+displays a colored button, but needs logic for further user interaction. Update `popup.js` by adding
+the following to the end of the file.
 
```

Comment:
Grammar: No comma needed for "displays a colored button but needs logic..."

Reply:
Resolved in next commit.